src/Button/Button.tsx

```
@@ -0,0 +1,168 @@
+}) => {
+    return (
+        <StyledButton
+            type="button"
+            variant={variant}
+            disabled={disabled || loading}
+            aria-disabled={disabled || loading}
+            {...rest}
+        >
+            {loading && (
+                <Spinner
+                    role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)